### DISCUSSION TITLE
Error: Nginx test failed

### DISCUSSION THREAD

**Discussion by iShariefi** [EasyEngine/easyengine#1607] [Support Requests]
The site was running fine. Suddenly it's getting a '502 Bad Gateway'. 

When I tried to do 'site reload', the output was, 
`Reloading nginx
Error response from daemon: Container b88a010669dada1e45bbb38d993499723c90e3e1e8f5258d2929114dbc5ae705 is restarting, wait until the container is running`.

What can I do now?

**Comment by dhsathiya** [comment]
Maybe a reboot or docker daemon restart will help you. I suggest a reboot.
